docs/api-reference/mapbox/overview.md

```diff
@@ -37,156 +27,39 @@ npm install @deck.gl/mapbox
 ```
 
 ```js
-import {MapboxLayer} from '@deck.gl/mapbox';
+import {MapboxOverlay} from '@deck.gl/mapbox';
 ```
 
 
-## Examples
-
-### Using with Pure JS
+## Use Cases
 
-To create a Mapbox-compatible deck.gl layer:
+One should note that this module is *not required* to use mapbox-gl as a base map for deck.gl. It is easier to understand the concepts of the module if you are already a mapbox-gl developer. Visit the [mapbox base map developer guide](/docs/developer-guide/base-maps/using-with-mapbox.md) for a complete list of your options.
```

Comment:
I think this could be confusing to developers. If the module is not required, what is the alternative? Link to an other approach?